packages/mobile/src/geth/networkConfig.ts

```diff
@@ -36,6 +44,8 @@ const networkConfigs: { [testnet: string]: NetworkConfig } = {
     blockchainApiUrl: 'https://integration-dot-celo-testnet.appspot.com/',
     pgpnpUrl: pgpnpUrlStaging,
     pgpnpPubKey: pgpnpPubKeyStaging,
+    peerDiscovery: USE_FULL_NODE_DISCOVERY,
+    bootnodeEnodes: BootnodeUtils.getBootnodes(Testnets.integration).split(';'),
```

Comment:
This is a config which we parse early in the app lifecycle. Given that getBootnodes is async, I don't think this is the right place for this call. Somewhere in `geth/saga.ts` or `geth/geth.ts` would be better

Reply:
Cool, Connor told me to move it to be similar to Static Nodes.